src/nodes/kolibri-broker/kolibri-broker.ts

```diff
@@ -49,6 +50,7 @@ const nodeInit: NodeInitializer = (RED): void => {
         this.broker = config.broker;
         this.port = config.port;
         this.project = config.project;
+        this.path = config.path;
```

Comment:
node-red seams not to pass the default values in here. So we need to ensure this has a default value e.g. '/' if the path is undefined.